### PR TITLE
Added Readme to project

### PR DESCRIPTION
**Pull request by tyseawood**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Tech Stack
+
+**Client:** TypeScript
+
+**Server:** Node, Express
+
+
+## Support
+
+For support, email tyseawood@protonmail.com.
```
Comment (DavidScales) [inline README.md:52]: it feels risky to leave an email in public (bots might spam you) but I've had my gmail in my profile forever and haven't noticed any issues. I guess whatever that is worth ha

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This is a movie search web application that allows you to search any movie within the database. This app also allows for adding movies to your favorites and storing them for later recall.
+
+
+## Run Locally
+
+Clone the project
+
+```bash
+  git clone https://link-to-project
```
[the remark this answers is not in the document]
Comment (tyseawood) [reply README.md:12]: I'll change the name on the repo and then update this.

File: README.md (hunk 1 above)
Comment (tyseawood) [reply README.md:52]: I agree with you, I can put a info@gmail.com or something here instead.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+Go to the project directory
+
+```bash
+  cd my-project
```
[the remark this answers is not in the document]
Comment (tyseawood) [reply README.md:18]: I'll update this and replace deployment "build" throughout

**Review by DavidScales — APPROVED** [review]

**Pull request merged**